pkg/altbn128/altbn128.go

```diff
@@ -60,8 +60,11 @@ func mod(i, m *big.Int) *big.Int {
 	return new(big.Int).Mod(i, m)
 }
 
-func modSqrt(i, m *big.Int) *big.Int {
-	return new(big.Int).ModSqrt(i, m)
+// modSqrt returns square root of x mod p if such a square root exists. The
+// modulus p must be an odd prime. If x is not a square mod p, function returns
+// nil.
```

Comment:
do we want to add that it panics if `p` is not odd?

Reply:
I am not sure, we disallow this value. I think we don't have to define what happens if a disallowed value is provided in this case. What do you think?

Reply:
ah, that makes sense.